R/parameters_table.R

```diff
@@ -78,6 +78,32 @@ parameters_table <- function(x, pretty_names = TRUE, stars = FALSE, digits = 2,
     x <- x[!names(x) %in% c(ci_low, ci_high)]
   }
 
+  # Other CIs
+  other_ci_low <- names(x)[grep("_CI_low", names(x))]
```

Comment:
If we can be sure (and make this a naming convention) that "other" CIs follow the pattern `<my_parameter_name>_CI_low`, you could add a `$` to the pattern, to ensure it only captures `_CI_low` and `_CI_high` at the end of a string;

```r
other_ci_low <- names(x)[grep("_CI_low$", names(x))]
```

same for `strsplit()`.

Reply:
> hat "other" CIs follow the pattern

Well, we can decide on that convention. As far as I know its first usage will be in this context of estimate + effect size

Reply:
@mattansb ?

Reply:
What is and is not captured by "other"?

Reply:
> What is and is not captured by "other"?

The intention is that all CI-columns _starting_ with `CI_low` or `CI_high` are processed by default. Then, all other CI columns that _end_ with `_CI_low` or `_CI_high` (like `Eta2_partial_CI_low`) are also processed (captured by "other").

Reply:
Sounds okay to me 👍